docs/DEVELOPER_GUIDE.md

```diff
@@ -54,11 +54,11 @@ You must install these tools:
 KFServing currently requires `Knative Serving` for auto-scaling, canary rollout, `Istio` for traffic routing and ingress.
 
 * You can follow the instructions on [Set up a kubernetes cluster and install Knative Serving](https://knative.dev/docs/install/) or
-[Custom Install](https://knative.dev/docs/install/knative-custom-install) to install `Istio` and `Knative Serving`. Observability plug-ins are good to have for monitoring. For working with KFServing master, we would recommend KNative 0.9.0+ and Istion 1.3.1+
+[Custom Install](https://knative.dev/docs/install/knative-custom-install) to install `Istio` and `Knative Serving`. Observability plug-ins are good to have for monitoring. For working with KFServing master, we would recommend KNative 0.11.1 and Istio 1.3.1
```

Comment:
I think this is redundant as already stated in the  `Prerequisites`, also a lot of users are not on Istio 1.3.1 yet.

Reply:
Done